README.md

```diff
@@ -70,11 +70,12 @@ export function configure(aurelia) {
      .then(() => {
       const registry = aurelia.container.get(CustomElementRegistry);
 
-      // with useView path strategy
+      // with useView path strategy & globalResource
       @useView('path/to/view.html')
       class MyCarousel {
         // ...
       }
+      aurelia.use.globalResources(MyCarousel);
```

Comment:
Should it be `registry.register(MyCarousel)`?

Reply:
Oh, I see the below changes. What is the intention of adding both of them? You wanna show it's possible to have both available in an application?

Reply:
In my tests, `registry.register(MyCarousel)` only works if the element has a defined behaviour (i.e., is registered first as a `globalResource`). I assumed this was intentional, so updated the docs accordingly.

Apologies if I have made the wrong choice!

Reply:
Did you return/await the register `Promise`? I think there could be something wrong, since it shouldn't need anything declared global at all. Webcomponent plugin can run independently